src/main/io/serial_cli.c

```diff
@@ -154,7 +154,7 @@ static void cliVersion(char *cmdline);
 static void cliRxRange(char *cmdline);
 static void cliPFlags(char *cmdline);
 
-#ifndef SKIP_TASK_STATISTICS
+#if !defined(SKIP_TASK_STATISTICS) && !defined(SKIP_CLI_RESOURCES)
```

Comment:
Why `resource` command gets disabled when there is a define for `SKIP_TASK_STATISTICS` ?